### ISSUE TITLE
Conversion from Bruker to Enhanced MR Image Storage fails

### ISSUE THREAD

**Issue by lamyj** [lamyj/dicomifier#1]
When converting Bruker to DICOM from the GUI for a subject with several studies, the output directory is not created and the conversion fails

```
terminate called after throwing an instance of 'boost::filesystem::filesystem_error'
    what():  boost::filesystem::directory_iterator::construct: No such file or directory
```

**Comment by lamyj** [comment]
This error is actually caused by the conversion of Bruker to Enhanced MR Image Storage: the sortPixelData function has been removed

**Comment by lamyj** [comment]
Closed by 62340ae
